opm/simulators/wells/ParallelWellInfo.hpp

```diff
@@ -161,6 +162,8 @@ class GlobalPerfContainerFactory
                            std::size_t num_components) const;
 
     int numGlobalPerfs() const;
+    int globalToLocal(const int globalIndex) const;
```

Comment:
Is here a reason that `int` and `std::size_t` are used for these two functions?

Reply:
Yes, Markus is right-